conda/recipes/libwholegraph/meta.yaml

```diff
@@ -41,19 +41,28 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
     - {{ compiler('cxx') }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} ={{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }}
+    {% endif %}
+    - cuda-version ={{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - cudatoolkit
+    {% else %}
+    - cuda-cudart-dev
+    - cuda-driver-dev
```

Comment:
I wonder why wholegraph needs CUDA driver library.

Reply:
Nevermind, just found out the source code that uses memory management functions from CUDA driver API.